### PR TITLE
Dropna functionality

### PR DESCRIPTION
**Pull request by kvnkho**
Added tests for it. It's passing on local for spark, dask, and native.

### CODE REVIEW

**Comment by goodwanghan** [comment]
fugue/execution/execution_engine.py     161      1    99%   354

L354 is the only place that is not covered by unit test, so the coverage dropped to 99%, can we write tests to cover that?

**Comment by kvnkho** [comment]
Yep should be able to cover easily.

File: fugue_test/builtin_suite.py [hunk 1]
```diff
@@ -465,6 +465,33 @@ def test_distinct(self):
                     )
                 )
 
+        def test_dropna(self):
```
Comment (goodwanghan) [inline fugue_test/builtin_suite.py:468]: I really like these tests. They are very readable, like an instruction of the function. Great job!

File: fugue_test/execution_suite.py [hunk 2]
```diff
@@ -444,6 +444,42 @@ def test_distinct(self):
                 throw=True,
             )
 
+        def test_dropna(self):
```
Comment (goodwanghan) [inline fugue_test/execution_suite.py:447]: Nice!

**Review by goodwanghan — APPROVED** [review]

**Pull request merged**